src/components/mission/Mission.js

```diff
@@ -18,14 +27,26 @@ const Mission = () => {
         </thead>
         <tbody>
           {missionsArray.map((item) => (
-            <tr key={item.mission_id}>
```

Comment:
You can destructure item to make code cleaner i.e : const { mission_id,... } = item;